django/contrib/admin/sites.py

```diff
@@ -470,7 +470,7 @@ def index(self, request, extra_context=None):
         app_dict = self._build_app_dict(request)
 
         # Sort the apps alphabetically.
-        app_list = list(six.itervalues(app_dict))
+        app_list = list(app_dict.values())
```

Comment:
Only because the very next line is .sort(), so I can replace the two with sorted()